plugins/CoreAdminHome/Commands/PurgeOldArchiveData.php

```diff
@@ -68,7 +69,12 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $archivePurger = $this->archivePurger;
+        $logger = null;
+        if ($output->getVerbosity() <= OutputInterface::VERBOSITY_NORMAL) {
+            $logger = new NullLogger();
+        }
```

Comment:
Ah yes I think we discussed that in slack! Could you maybe add a comment to explain it?